backend/worker/worker.go

```diff
@@ -784,6 +784,12 @@ func (w *Worker) processSession(ctx context.Context, s *model.Session) error {
 
 	pagesVisited := len(visitFields)
 
+	workspace, err := w.Resolver.GetWorkspace(project.WorkspaceID)
+	if err != nil {
+		return err
+	}
+	withinBillingQuota, _ := w.PublicResolver.IsWithinQuota(ctx, pricing.ProductTypeSessions, workspace, time.Now())
```

Comment:
do we still need the check in `InitializeSessionImpl` ?

Reply:
I think it's better to have the check there too, just because otherwise users will be able to see all their live sessions even though they're not paying